lib/Model/Importer/Binary/ELFImporter.cpp

```diff
@@ -224,6 +224,7 @@ Error ELFImporter<T, HasAddend>::import(const ImporterOptions &Options) {
 
       auto MaybeSectionName = TheELF.getSectionName(SectionHeader);
       if (auto Error = MaybeSectionName.takeError()) {
+        // TODO: is there anything useful we can extract from this error to log?
```

Comment:
Yes, we should *never* silently suppress errors! Log!

Reply:
Ah, my bad.